mmv1/products/integrations/Client.yaml

```diff
@@ -83,18 +84,34 @@ properties:
           the kms key is stored at the same project as customer's project and ecrypted
           with CMEK, otherwise, the kms key is stored in the tenant project and
           encrypted with GMEK.
+    conflicts:
+      - provision_gmek
   - !ruby/object:Api::Type::Boolean
     name: 'createSampleWorkflows'
     description: |
       Indicates if sample workflow should be created along with provisioning.
     immutable: true
     ignore_read: true
+    deprecation_message: "`create_sample_workflows` is deprecated and will be removed in a future major release. Use `create_sample_integrations` instead."
+    conflicts:
+      - create_sample_integrations
+  - !ruby/object:Api::Type::Boolean
+    name: 'createSampleIntegrations'
+    description: |
+      Indicates if sample integrations should be created along with provisioning.
+    immutable: true
+    ignore_read: true
+    conflicts:
+      - create_sample_workflows
   - !ruby/object:Api::Type::Boolean
     name: 'provisionGmek'
     description: |
       Indicates provision with GMEK or CMEK.
+    deprecation_message: "`provision_gmek` is deprecated and will be removed in a future major release. Client would be provisioned as gmek if `cloud_kms_config` is not given."
```

Comment:
@SarahFrench The field is being marked as deprecated. The public docs are being updated however, discovery reflect the field deprecation.
https://integrations.googleapis.com/$discovery/rest?version=v1

Reply:
Sounds good, thanks for providing that link